Dataframe structure

duttob opened the pull request:
Ajout de la structure du Dataframe global (Structure DataFrame + Series et méthodes de création par csv et manuelle). 
Permet d'afficher les Dataframes ainsi que les Series.
Implémente les fonctionnalités de typage automatique pour les csv.

Benjamin100 committed:
First version of the Dataframe structure :Adding the dataframe structure and its graphical representation in the console.

Benjamin100 committed:
Bug fixing and adding tests

Benjamin100 committed:
Fixing CSV load : Changing the type of columns (series), previously it was multi typed now each series as a unique type that is determined while parsing the csv.

hugorouillard commented on cities.csv:
le fichier n'est utilisé ni dans le code ni dans les tests -> on peut le supprimer

hugorouillard commented on src/main/java/org/example/Dataframe.java:
```diff
@@ -0,0 +1,132 @@
+ * Represents a dataframe structure containing multiple series of data.
+ */
+public class Dataframe {
+    public Serie<?>[] data_tab;
```
peut être mettre le field en private et ajouter un getter à la place pour que ce soit plus propre?

hugorouillard commented on src/main/java/org/example/Serie.java:
Le singulier de "Series" en anglais c'est aussi "Series" donc le nom de la classe est un peu bizarre. Ça serait mieux de renommer la classe en `Series` pour rester cohérent je pense.

hugorouillard commented on test1.csv:
même chose, le fichier n'est utilisé nulle part donc on peut le supprimer pour nettoyer un peu le repo

hugorouillard reviewed (commented):
Globalement le code et les tests sont bien, juste 2-3 changements par rapport à la structure.

Benjamin100 committed:
Fixing problems after code review of the pull request:
- Refactoring "Serie" as Series
- Making dataframe "data_tab" var private
- Deleting useless files
